.editorconfig

```diff
@@ -4,7 +4,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Oh no... Me and airbnb - we use 2 spaces cause why do it in 4 what you can do in 2 - 
https://github.com/airbnb/javascript#whitespace